internal/shipwright/build/build.go

```diff
@@ -68,7 +68,7 @@ func (sb *ShipwrightBuild) CreateOrUpdate(ctx context.Context, owner client.Obje
 
 	return ctrl.CreateOrUpdate(ctx, sb.Client, object, func() error {
 		object.Spec = shipwrightv1alpha1.ShipwrightBuildSpec{
-			TargetNamespace: common.OpenShiftBuildNamespaceName,
+			TargetNamespace: common.FetchCurrentNamespaceName(),
```

Comment:
This function does a disk read in every reconcile for something that we don't expect to change in the container's lifetime. I recommend this is called once in `cmd/main.go` and passed to the reconciler as a parameter/struct variable.

Reply:
++1